script/test/utils.sh

```diff
@@ -42,6 +43,15 @@ runtime_type = "${CONTAINERD_RUNTIME}"
 EOF
   fi
   CONTAINERD_CONFIG_FILE="${config_file}"
+
+  FAILPOINT_CONTAINERD_RUNTIME="runc-fp.v1"
+
+  # Add runtime with failpoint
+  cat << EOF | tee -a "${CONTAINERD_CONFIG_FILE}"
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc-fp]
+  pod_annotations = ["io.containerd.runtime.v2.shim.failpoint.*"]
+  runtime_type = "${FAILPOINT_CONTAINERD_RUNTIME}"
+EOF
 fi
```

Comment:
Do you need to add CNI config like `"${FAILPOINT_CNI_CONF_DIR}/10-containerd-net.conflist"`?

Reply:
in release/1.5, we don't have the feature about supporting CNI config per runtime. So in this patch, I introduce the update-cni-config on-demand.